nucypher/cli/commands/stake.py

```diff
@@ -130,6 +129,7 @@
 from nucypher.config.characters import StakeHolderConfiguration
 from nucypher.utilities.gas_strategies import construct_fixed_price_gas_strategy
 
+option_csv_file = click.option('--csv-file', help="Filepath for CSV output", type=click.Path(dir_okay=False))
```

Comment:
Consider a two flag approach, one using the a default, generated filepath, and the other to customize it.

Reply:
✔️